### PR TITLE
[IMP] stock_request_picking_type: Use Submitted state

### PR DESCRIPTION
**Pull request by max3903**
#595 

- [x] Depends on #675

### CODE REVIEW

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

Please update module version

**Review by patrickrwilson — APPROVED** [review]
I've got now suggested changes.

**Comment by max3903** [comment]
@osi-scampbell Please implement the onchange method to update the location based on the direction.
See https://github.com/OCA/field-service/blob/a1e3a506d83f922b024d4e0199076a31a54424b5/fieldservice_stock/models/stock.py#L14

File: stock_request/models/stock_request.py [hunk 1]
```diff
@@ -110,6 +110,17 @@ def _get_default_requested_by(self):
          'Stock Request name must be unique'),
     ]
 
+    @api.onchange('direction')
+    def _onchange_location_id(self):
+        if self.direction == 'outbound':
+            # Stock Location set to the Stock Location of the Order
```
Comment (max3903) [inline stock_request/models/stock_request.py]:
```
# Stock Location set to Partner Locations/Customers
```

**Review by bodedra — APPROVED** [review]
LGTM :+1: 

Manor Comment:

Please rebase PR branch and make CI happy.

**Comment by bodedra** [comment]
@max3903 I have taken care of it and make CI happy.

@rousseldenis Could you please update your review ?

**Pull request merged**